lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -1014,8 +1017,13 @@ setReadOnly( Complex64Array.prototype, 'indexOf', function indexOf( searchElemen
 		throw new TypeError( format( 'invalid argument. First argument must be a complex number. Value: `%s`.', searchElement ) );
 	}
 	if ( arguments.length > 1 ) {
-		if ( !isNonNegativeInteger( fromIndex ) ) {
-			throw new TypeError( format( 'invalid argument. Second argument must be a nonnegative integer. Value: `%s`.', fromIndex ) );
+		if ( !isInteger( fromIndex ) ) {
+			throw new TypeError( format( 'invalid argument. Second argument must be an integer. Value: `%s`.', fromIndex ) );
+		} else if ( fromIndex < 0 ) {
```

Comment:
```suggestion
		}
		if ( fromIndex < 0 ) {
```

Reply:
No need for the `else` here. If L1020 evaluates to `true`, by 1021 no further code is accessible.